package.json

```diff
@@ -53,7 +53,7 @@
     "agentkeepalive": "^4.0.2",
     "applicationinsights": "^1.4.0",
     "applicationinsights-native-metrics": "^0.0.5",
-    "audit-ci": "^2.4.2",
+    "audit-ci": "^3.0.0",
```

Comment:
the breaking change from 2.x.x to 3.x.x is dropping support for node 6, which doesn't affect us.